src/tools/react-native.ts

```diff
@@ -309,10 +309,9 @@ export function updateExpoRouterSrcDir(toolbox: GluegunToolbox) {
   const TARGET_DIR = filesystem.path(process.cwd())
   const expoRouterFilesToFix = [
     "tsconfig.json",
+    // has its own tsconfig, needs updating separately
     "test/i18n.test.ts",
-    "src/devtools/ReactotronConfig.ts",
-    "src/components/Toggle/Switch.tsx",
-    "src/components/ListView.tsx",
+    "test/setup.ts",
```

Comment:
Curious: Why are we removing the 3 lines on the left?

Reply:
Because the alias was updated so there are less specific changes to files that are necessary. Before the changes were very specific and didn't scale well, now @lindboe has adjusted the path alias in `tsconfig` and we don't have to do so many transformations across the source files (whether they're in `app/` or `src/`) - less maintenance for us.

Reply:
Thanks!

Reply:
In addition, the test files are still in there because there's a separate TS config, but if we can unify those better we can also update the test files automatically. But I'm planning on circling back to that in a later change.